rust/arrow/src/array/null.rs

```diff
@@ -134,13 +130,6 @@ mod tests {
         assert_eq!(null_arr.len(), 32);
         assert_eq!(null_arr.null_count(), 32);
         assert_eq!(null_arr.is_valid(0), false);
-
-        assert_eq!(0, null_arr.get_buffer_memory_size());
```

Comment:
same question here about why does the PR remove the test coverage rather than updating it?